include/klee/ExecutionState.h

```diff
@@ -162,7 +163,9 @@ class ExecutionState {
   ExecutionState() : ptreeNode(0) {}
 
 public:
-  ExecutionState(Executor *executor, KFunction *kf, bool enableNvmHeuristic=false);
+  ExecutionState(Executor *executor, 
+                 KFunction *kf, 
+                 const Interpreter::ModuleOptions &modOpts);
```

Comment:
The assumption behind suggesting a command line option was that it would remove this constructor parameter. I thought these `cl::opt` things were supposed to be globally accessible. I know there are other source files that declare `extern cl::opt` in their header files and get the options that way.